google/cloud/security/common/data_access/sql_queries/select_data.py

```diff
@@ -149,9 +149,25 @@
     WHERE project_number = {1};
 """
 
+# TODO: reduce these sql to a generic statement
+SELECT_POLICY_VIOLATIONS = """
+    SELECT * FROM violations_{0}
+    WHERE violation_type = 'ADDED' 
+    OR violation_type = 'REMOVED';
```

Comment:
Can we change this ```OR``` to use a list?
```WHERE violation_type in ['ADDED', 'REMOVED']```
https://stackoverflow.com/a/8945575/2830207